packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.4
```

Comment:
This PR doesn't add the platform interface, it merely updates the homepage in the pubspec since you moved the directory

Reply:
Ah true. Then I don't get why we add a new version at all. Does this package need to get published? Only `video_player_platform_interface` needs to be, right?

Reply:
It's very minor, but if you don't publish there will be a period of time where the 'Homepage' link on `pub.dev` for `video_player` will point to the wrong place.

Reply:
Oh I got it.